content/docs/faq-functions.md

```diff
@@ -295,3 +301,58 @@ class Searchbox extends React.Component {
   }
 }
 ```
+
+#### `requestAnimationFrame` throttling
+
+[`requestAnimationFrame`](https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame) is a way of queuing a function to be executed in the browser at the optimal time for rendering performance. It is very similar to `setTimeout` except it takes no time argument. A function that is queued with `requestAnimationFrame` will fire in the next frame. The browser will work hard to ensure that there are `60` frames per second (`60 fps`). However, if the browser is unable to it will naturally *limit* the amount of frames in a second. For example, a device might only be able to handle `30 fps` and so you will only get `30` frames in that second. Using `requestAnimationFrame` for throttling is a useful technique in that it prevents you from doing more than `60 updates` in a second. If you are doing `100` updates in a second this creates additional work for the browser that the user will not see anyway.
```

Comment:
I'm not sure about this sentence:
> It is very similar to `setTimeout` except it takes no time argument.

`requestAnimationFrame` is only similar in that it executes code asynchronously. Maybe we should just remove this sentence to avoid potentially confusing people?

Reply:
yeah - or i'll see if i can get it a bit clearer